build-system/app-utils.js

```diff
@@ -57,34 +103,8 @@ const replaceUrls = (mode, file, hostName, inabox, storyV1) => {
       file = file.replace(/\/dist\/amp\.js/g, filename);
     }
   } else if (mode == 'compiled') {
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\.js/g,
-      hostName + '/dist/v0.js'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/shadow-v0\.js/g,
-      hostName + '/dist/shadow-v0.js'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.js/g,
-      hostName + '/dist/amp4ads-v0.js'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/video-iframe-integration-v0\.js/g,
-      hostName + '/dist/video-iframe-integration-v0.js'
-    );
-    file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\/(.+?).js/g,
-      hostName + '/dist/v0/$1.js'
-    );
-    file = file.replace(
-      /\/dist\/v0\/examples\/(.*)\.max.js/g,
-      '/dist/v0/examples/$1.js'
-    );
-    file = file.replace(
-      /\/dist.3p\/current\/(.*)\.max.html/g,
-      hostName + '/dist.3p/current-min/$1.html'
-    );
+    file = replaceCompiledUrls(file, hostName);
+
     if (inabox) {
```

Comment:
@lannka are there any `inabox` URL replacements that need to be done to use RTV urls?

Reply:
thanks for asking. `amp4ads-v0.js` needs to be versioned.
amp4ads-lite-v0.js is obsolute, pls just remove